Guidelines.md

```diff
@@ -1709,11 +1709,9 @@ All service URLs must be HTTPS (that is, all inbound calls MUST be HTTPS). Servi
 
 We recommend that services that allow client defined Web Hook Callback URLs SHOULD NOT transmit data over HTTP. This is because information can be inadvertently exposed via client, network, server logs and other mechanisms.
 
-However, there are scenarios where the above recommendations cannot be followed due to client endpoint or software limitations.  Consequently, services MAY allow web hook URLs that are HTTP.  
+However, there are scenarios where the above recommendations cannot be followed due to client endpoint or software limitations.  Consequently, services MAY allow web hook URLs that are HTTP. Another example is that services may not want to require developers to generate certificates during onboarding. In this case, services might allow HTTP web hook URLs to be used on test accounts.
```

Comment:
I'll change it back because I'm not that bothered by it; the information is still readable and correct. I just didn't the usage of `For example`. That would usually be used to give an example after explaining something, but, in this case, it just seems to random.